src.gen/@amzn/codewhisperer-streaming/dist-cjs/CodeWhispererStreaming.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
```

Comment:
why is `/dist-cjs/` being committed? surely we don't need both the `.js` files and the `.ts` files?